src/components/SectionBox/useStyles.ts

```diff
@@ -9,7 +9,7 @@ const useStyles = () => {
       justifyContent: 'center',
       minWidth: '272px',
       '&.sectionbox__main': {
-        padding: theme.spacing(3),
+        padding: theme.spacing(3,0),
```

Comment:
```suggestion
        padding: theme.spacing(3, 0),
```
suggest to add space here, to have a consistent style

Reply:
@teamchong how about `theme.spacing(3, 1)`?

Reply:
all good for me, so the previous `theme.spacing(3,0)` is typo?

Reply:
No, was focusing on the left and right padding, and the title has a top padding of 24px so thought `(3, 0)` is appropriate.